dashboard/app/models/school_district.rb

```diff
@@ -35,7 +39,7 @@ def self.get_seed_filename(stub_school_data)
   # Seeds all the data from the source file.
   # @param options [Hash] Optional map of options.
   def self.seed_all(options = {})
-    options[:stub_school_data] ||= CDO.stub_school_data
+    options[:stub_school_data] = CDO.stub_school_data unless options.key?(:stub_school_data)
```

Comment:
I think there's no way to use seed_all manually with `stub_school_data` set to false as-is, because ||= will always use `CDO.stub_school_data`